### DISCUSSION TITLE
suspicious.log

### DISCUSSION THREAD

**Discussion by maestropss** [DNSCrypt/dnscrypt-proxy#2170] [Q&A]
Hallo. I see the new file suspicious.log after the upgrade to dnscrypt-proxy-win64-2.1.2. Can't find any info about it. What is the purpose of it? Thanks

**Comment by lifenjoiner** [comment]
There is no such `suspicious.log` file by default. It should depend on your settings.
Related:
https://github.com/DNSCrypt/dnscrypt-proxy/blob/e124623ffced33c372f257071aa0949af03b37a5/dnscrypt-proxy/example-dnscrypt-proxy.toml#L481-L493
